deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -1370,6 +1370,13 @@ fun(Conf) ->
     end
 end}.
 
+%% Register node during cluster formation when backend supports registration.
+%%
+
+{mapping, "cluster_formation.registration", "rabbit.cluster_formation.register", [
```

Comment:
The key used in `rabbit_peer_discovery` is `registration`, not `register`. I will correct this on your behalf.